test/any.specs.ts

```diff
@@ -0,0 +1,158 @@
+import "mocha"
+import fc, { Value } from "fast-check"
+import { areEquals, isEmpty, isNotEmpty } from "../src/any"
+
+//isEmpty tests
+describe("isEmpty: ", () => {
+  it("Should return true for empty strings", () =>
+    fc.assert(
+      fc.property(fc.string({ maxLength: 0 }), (value) =>
+        isEmpty(value)
+      )
+    ))
+
+  it("Should return false for non-empty strings", () =>
+    fc.assert(
+      fc.property(fc.string({ minLength: 1 }), (value) => !isEmpty(value)
+      )
+    ))
+
+  it("Should return true for empty arrays", () =>
+    fc.assert(
+      fc.property(fc.array(fc.nat(), { maxLength: 0 }), (value) =>
+        isEmpty(value)
+      )
+    ));
+
+  it("Should return false for non-empty arrays", () =>
+    fc.assert(
+      fc.property(fc.array(fc.nat(), { minLength: 1 }), (value) => {
+        return !isEmpty(value)
+      })
+    ))
+
+  it("Should return true for empty objects", () =>
+    fc.assert(
+      fc.property(fc.object({ maxKeys: 0 }), (value) => {
+        isEmpty(value)
+      })
+    ))
+
+  it("Should return false for non-empty objects", () =>
+    fc.assert(
+      fc.property(fc.object({ maxKeys: 200 }), (value) => {
```

Comment:
plutôt minKeys que maxKeys je pense

Reply:
J'ai pas de minKeys c'est pour ça j'ai utilisé maxKeys :/

Reply:
Ok, it does not exist and after checking there is no way of limiting the number of keys either.
But what you do here isn't valid, you are trying to test a non-empty object while the arbitrary can generate an empty object. In those cases, when you have no other choice, the filter function is there for you.

```ts
fc.object().filter(obj => Object.keys(obj).length > 0)
```